server/src/db/migrations/20231031140921_createDestinations.cjs

```diff
@@ -0,0 +1,26 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("destinations", (table) => {
+        table.bigIncrements("id")
+        table.string("name").notNullable()
+        table.text("description")
+        table.string("location").notNullable()
+        table.string("destinationType").notNullable()
```

Comment:
instead of repeating the table name `destination` in the column name I'd recommend calling it `type` so that you can use `destination.type` as opposed to `destination.destinationType`